solr/parser.go

```diff
@@ -34,6 +34,84 @@ func (parser *FireworkResultParser) Parse(resp *[]byte) (FireworkSolrResult, err
 	return res, err
 }
 
+type ExtensiveResultParser struct {
```

Comment:
It will contain both search result and facets.

On Monday 30 May 2016 12:55 PM, Nguyen Van Nhu wrote:
>
> In solr/parser.go 
> <https://github.com/vanng822/go-solr/pull/38#discussion_r65033592>:
>
> > @@ -34,6 +34,84 @@ func (parser *FireworkResultParser) Parse(resp *[]byte) (FireworkSolrResult, err
> >  	return res, err
> >  }
> >
> > +type ExtensiveResultParser struct {
>
> Wouldn't it be better to name it as it's task, like FacetResultParser
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/vanng822/go-solr/pull/38/files/47fc826202fabe98d97262ab6de2e9fbced42913#r65033592>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ASRvIWNp4-Grk3vzLwwOuLovZNkKSq0Iks5qGpD9gaJpZM4Ipd3V>.
>


-- 

------------------------------
IMPORTANT: This is an e-mail from HiFX IT Media Services Pvt. Ltd. Its 
content are confidential to the intended recipient. If you are not the 
intended recipient, be advised that you have received this e-mail in error 
and that any use, dissemination, forwarding, printing or copying of this 
e-mail is strictly prohibited. It may not be disclosed to or used by anyone 
other than its intended recipient, nor may it be copied in any way. If 
received in error, please email a reply to the sender, then delete it from 
your system. 

Although this e-mail has been scanned for viruses, HiFX cannot ultimately 
accept any responsibility for viruses and it is your responsibility to scan 
attachments (if any).

​
Before you print this email or attachments, please consider the negative 
environmental impacts associated with printing.